src/components/Text/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+
+//functional component
+
+const Text = (props) => {
+   //const { text } = props
+   const text = props.text
```

Comment:
@AnthoniaOkafor, can we use ES6 please, something like `const { text } = props;` because part of the goal here is to also get you to be familiar with ES6 syntax.

Reply:
I thought any one of them is ES6. I will make the changes